ui/app/components/gas-customization/gas-modal-page-container/gas-modal-page-container.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import PageContainer from '../../page-container'
+import { Tabs, Tab } from '../../tabs'
+
+export default class GasModalPageContainer extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    hideModal: PropTypes.func,
+  }
+
+  state = {}
+
+  renderBasicTabContent () {
+    return (
+      <div className="basic-tab"/>
+    )
+  }
+
+  renderAdvancedTabContent () {
+    return (
+      <div className="advanced-tab"/>
+    )
+  }
+
+  renderInfoRow (className, totalLabelKey, fiatTotal, cryptoTotal) {
+    return (
+      <div className={className}>
+        <div className="total-info">
+          <span className="total-label">{`${this.context.t(totalLabelKey)}:`}</span><span className="total-value">{fiatTotal}</span>
+        </div>
+        <div className="sum-info">
+          <span className="sum-label">{`${this.context.t('amountPlusTxFee')}`}</span><span className="sum-value">{cryptoTotal}</span>
+        </div>
+      </div>
+    )
+  }
+
+  renderTabContent (mainTabContent) {
+    return (
+      <div className="gas-modal-content">
+        { mainTabContent() }
+        { this.renderInfoRow('info-row--fade', 'originalTotal', '$20.02 USD', '0.06685 ETH') }
```

Comment:
Nice way of avoiding duplication of the info rows. This is a limitation of `PageContainer` that I should add support for.